evalml/pipelines/utils.py

```diff
@@ -676,8 +687,8 @@ def generate_pipeline_code(element, features_path=None):
     pipeline_code = "\n".join(code_strings)
     if has_dfs_and_features:
         pipeline_code = pipeline_code.replace(
-            "'DFS Transformer':{},",
-            "'DFS Transformer':{'features':features},",
+            "DFS Transformer':{},",
+            "DFS Transformer':{'features':features},",
```

Comment:
Was this on purpose? If not the single quote might be misaligned

Reply:
Yes - this is to make it replace any pipelines with subpipelines (ensembling including) as the naming scheme would be "Linear Pipeline - DFS Transformer....'

Reply:
I'll write a comment about it.